Configuration.hpp

```diff
@@ -32,22 +32,22 @@
 // Include the current software version.
 #include "Version.h"
 
-// Include the user-specific local configuration
-#if __has_include("Configuration_local.hpp")
-    #include "Configuration_local.hpp"
-#elif __has_include("Configuration_local_CI.hpp")                                   // CI environment on GitHub
-    #include "Configuration_local_CI.hpp"
-#endif
-
 /**
  * Use default values for any parameters the user didn't provide.
  */
 
 // If board is not defined in local configuration, assume Arduino Mega2560
 #ifndef BOARD
-  #define BOARD BOARD_AVR_MEGA2560
+  // #define BOARD BOARD_AVR_MEGA2560
 #endif
 
+#ifndef BOARD
+  #error You have to specify the board
+#endif
+
+// Include the user-specific local configuration
+#include "LocalConfiguration.hpp"
```

Comment:
Yet another configuration renaming... users are going to love us.
[Later] Ahh, I see this is just an extra level.

Reply:
Yeah this is just to isolate all the ifdefs of the main configuration file and keep it cleaner